interface/super/edit_list.php

```diff
@@ -1064,6 +1094,49 @@ function mysubmit() {
                     </select>
                 </div>
 
+                <!--Added filter-->
+                <script type="text/javascript">
+                    function lister() {
+                        var urlFull   = new URL($(location).attr('href'));
+                        var list_from = parseInt($("#list-from").val());
+                        var list_to   = parseInt($("#list-to").val());
+                        var list_id_container = $("#list_id").val();
+
+                        if( list_from > list_to ){
+                            alert(<?php echo xlj("Please enter a enter valid range"); ?>);
+                            return false;
+                        }
+                        if( parseInt(list_from) >= 0 ){
+                            urlFull.searchParams.delete("list_from");
+                            urlFull.searchParams.set("list_from",list_from);
+                        }
+
+                        if( parseInt(list_from) >= 0 ){
```

Comment:
@yurige-matrix , This if block and the above if block use the same conditional. Can we combine them?

Reply:
I have already combined